### ISSUE TITLE
Duplicate version in CHANGELOG.md

### ISSUE THREAD

**Issue by regseb** [htmlhint/HTMLHint#699]
**Is your feature request related to a problem? Please describe.**
Since _0.15.2_, the changes of the releases are inserted in duplicate in [CHANGELOG.md](https://github.com/htmlhint/HTMLHint/blob/8918d96260838ed161f92366c2dc8d8873e726a6/CHANGELOG.md):

- [_chore(release): 0.15.2 [skip ci]_](https://github.com/htmlhint/HTMLHint/commit/dee8ea27615830ee85998be7101340524f0a7f09#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed)
- [_chore(release): 0.16.0 [skip ci]_](https://github.com/htmlhint/HTMLHint/commit/8918d96260838ed161f92366c2dc8d8873e726a6#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed)

**Describe the solution you'd like**
Add only once the changes for future versions. And remove duplicates for versions _0.15.2_ and _0.16.0_.

**Comment by regseb** [comment]
@coliff @thedaviddias Version _0.16.1_ are added in duplicate: https://github.com/htmlhint/HTMLHint/commit/ec2da2c0dd9c11c74a4a31629495c98997dc88f9#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed

**Comment by coliff** [comment]
yep, I made a new PR for it.. thanks!
